hack/ci/vagrant-helper.sh

```diff
@@ -22,10 +22,8 @@ if [ ! -f "$SSH_CONFIG" ]; then
   vagrant ssh-config > "$SSH_CONFIG"
 fi
 
-sudo="sudo"
-if [ "$ROOTLESS" = "rootless" ]; then
-  sudo=
+if [ "$ROOTLESS" != "rootless" ]; then
+  sudo="sudo"
```

Comment:
this may be breaking the github rootfull jobs somehow

Reply:
rewrote this